yarn.lock

```diff
@@ -380,10 +380,8 @@
   resolved "https://registry.yarnpkg.com/@discoveryjs/json-ext/-/json-ext-0.5.7.tgz#1d572bfbbe14b7704e0ba0f39b74815b84870d70"
   integrity sha512-dBVuXR082gk3jsFp7Rd/JI4kytwGHecnCoTtXFb7DB6CNHp4rg5k1bhg0nWdLGLnOV71lmDzGQaLMy8iPLY0pw==
 
-"@eppo/js-client-sdk-common@2.2.2":
+"@eppo/js-client-sdk-common@file:../js-client-sdk-common":
```

Comment:
This will need to be updated to either the git repo link after [this PR](https://github.com/Eppo-exp/js-client-sdk-common/pull/46) is merged or the published version is available in npm. For now, please ignore the failing automated checks